codeshifu/fast-food-fast
# #161276416 Add restaurant menu

### What does this PR do?
Implements logic for creating/adding a menu by admin.

#### How should this be tested?
- login as admin
- visit `add-menu.html`
- add restaurant menu
- visit index page to see the newly created menu

#### What are the relevant pivotal tracker stories?
[#161276416](https://www.pivotaltracker.com/story/show/161276416)
